### PR TITLE
Jrercoli/1a

### PR DESCRIPTION
**Pull request by jrercoli**
Start code review for this feature (**story 1a**)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# import sys
+import json
+filename = "fake_cars.txt"
```
Comment (jrercoli) [inline app.py:3]: put off filename from here and put on a context variable

File: app.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            car_id += 1
+
+    response = {
+        "statusCode": 200,
+        "headers": {},
+        "body": json.dumps(out_dict)
+    }
+
+    return response
+    # return 'Hola desde AWS lambda con Python' + sys.version
```
Comment (jrercoli) [inline app.py:53]: delete this line !

Comment (jrercoli) [reply app.py:53]: ok fine, i delete that

**Review by jrercoli — COMMENTED** [review]
You must resolve the changes

File: app.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    # fields in sample cars file
+    car_fields = ['company', 'model']
+
+    # output dict
+    out_dict = {}
+
+    # get the json event body, car requested
+    print(event)
+    # request_body = json.loads(event["body"]) if event.get("body") else json.loads(event)
+    if event.get('car'):
```
Comment (jrercoli) [inline app.py:15]: '''if .....
else

File: app.py (hunk 1 above)
Comment (jrercoli) [inline app.py:3]:
```suggestion
filename = env.var1
```

File: app.py (hunk 3 above)
Comment (jercolinanlabs) [reply app.py:15]:
```suggestion
    if event.get('door'):
```

**Review by jercolinanlabs — APPROVED** [review]
all code is ok

**Pull request merged**